File: GooglePlusblockedList.dart
import 'dart:async';
import 'dart:io';

void main() async {
  var res = await new File('list.html').readAsString();
  var match = new RegExp(r'已屏蔽.*?</div>');
  print('已屏蔽 ${match.allMatches(res).length} 人：');
  match.allMatches(res).map((m) => m[0])
      .forEach((f) => print(f.substring(3, f.indexOf('</div>'))));
}

File: GooglePlusblockedList.go
package main

import (
	"fmt"
	"io/ioutil"
	"regexp"
)

func main() {
	res, err := ioutil.ReadFile("list.html")
	if err != nil {
		fmt.Println(err)
	}
	exp := regexp.MustCompile("已屏蔽.*?</div>")
	match := exp.FindAllString(string(res), -1)
	fmt.Println("已屏蔽 ", len(match), " 人：")
	for _, v := range match {
		r := []rune(v)
		fmt.Println(string(r[3 : len(r)-6]))
	}
}
